src/styles/index.scss

```diff
@@ -1,3 +1,234 @@
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --square-background: #eee;
+  --square-width: 100px;
+  --square-height: 100px;
+  --square-border: 1px solid black;
+  --square-font: Arial, Helvetica, sans-serif;
+  --square-size: 30px;
+  --square-margin-right: 1px;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+ul,
+li {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.calendar__block {
+  display: flex;
+  flex-wrap: wrap;
+  width: 714px;
+  text-align: center;
+  line-height: 100px;
+  transition-duration: 0, 5s;
+}
+
+.calendar__day {
+  width: var(--square-width);
+  height: var(--square-height);
+  background: var(--square-background);
+  border: var(--square-border);
+  margin: var(--square-margin-right);
+}
+
+.calendar__block .calendar__day:first-child::before {
```

Comment:
Lucas, you don't need to use this to add the numbers, you can add them inside the HTML.
Use this css `content` only when needed, it polutes your code